### PR TITLE
Remove example configuration ini files

### PR DESCRIPTION
**Pull request by mwperina**
Example ini configuration files are no longer necessary and standard
configuration files should be used instead:

1. Fixes `README.developer.md` to not use example ini files
2. Removes `doc/example-hirte*.ini` files

Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

**Review by Yarboa — APPROVED** [review]
LGTM

File: README.developer.md [hunk 1]
```diff
@@ -126,15 +126,15 @@ nc <host> <port>
 A node can be run via:
 
 ```bash
-./bin/hirte-agent -n <agent name> -H <host> -P <port>
+./bin/hirte-agent -n <agent name> -H <host>
```
Comment (engelmi) [inline README.developer.md:129]: Why did you remove the port? This is still a supported cli option as far as I know

Comment (mwperina) [reply README.developer.md:129]: It is, but the port is set by default, so no need to set it

**Comment by alexlarsson** [comment]
lgtm

**Review by alexlarsson — APPROVED** [review]
